core/src/test/java/io/questdb/griffin/TruncateTest.java

```diff
@@ -112,15 +112,15 @@ public void testTruncateWithColumnTop() throws Exception {
                                     ") timestamp (k) partition by day"
                     );
 
-                    compile("alter table testTruncateWithColumnTop add column colum_with_top int");
+                    compile("alter table testTruncateWithColumnTop add column column_with_top int");
 
                     compile(
                             "insert into testTruncateWithColumnTop " +
                                     "select" +
                                     " cast(x as int) i," +
                                     " rnd_symbol('msft','ibm', 'googl') sym," +
                                     " timestamp_sequence('1970-01-01T12', 10000) k," +
-                                    " x as colum_with_top " +
+                                    " x as column_with_top " +
```

Comment:
I'm assuming this is a test and it doesn't care

Reply:
this particular one doesn't care